Typings:  Add support for Typings

mdavis777 opened the pull request:
This commit adds support for the typescript definition manager typings
https://github.com/typings/typings

mdavis777 commented:
I needed to add typings support for my current product.  I don't know if you want it or not.  If I need to make any changes to conform to any standards let me know.

imyller commented:
Thank you for your contribution @mdavis777

As I have no experience working with Typings, it's going to take a while to review this.

Feedback from others interested is very much welcome too.

As a side note, I'm currently considering a separation of core Node.js runtime recipes from helper recipes/classes to recipes-contrib or similar. Eventually I won't have enough resources to maintain every build-system specific class in the Node.js core layer.

mdavis777 commented:
I am in no hurry so take all the time you need. If it helps almost all of the work is straight out of the bower recipe and class and renamed.  
I updated the readme with relevant information in a second commit
